src/TaskParameters.ts

```diff
@@ -7,19 +7,18 @@ var fs = require('fs');
 
 export default class TaskParameters {
     // image builder inputs
-    public resourceGroupName: string;
+    public resourceGroupName: string = "";
     public location: string = "";
     public imagebuilderTemplateName: string;
     public isTemplateJsonProvided: boolean = false;
     public templateJsonFromUser: string = '';
-    public nowaitMode: string;
```

Comment:
What happened to no-wait-mode? Did you remove it because we are not supporting it?

Reply:
yaa I removed it because we are not supporting it for now